pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -383,6 +372,17 @@ func (l *podNIC) getPhase2Binding(domain *api.Domain) (BindMechanism, error) {
 	return nil, fmt.Errorf("Not implemented")
 }
 
+func retrieveMacAddress(iface *v1.Interface) (*net.HardwareAddr, error) {
```

Comment:
Now the method is not an inner method, I would change its name to -retrieveMacAddressFromVmiSpecIface.
I would also change the name of the parameter to vmiSpecIface, up to you, if you will change the method name maybe it will be clear enough what is this iface.

Reply:
Done.